server/controllers/task.js

```diff
@@ -0,0 +1,21 @@
+import pool from "../db/setup.js";
+
+export default class TaskController {
+  static createTask = async (body) => {
+    const client = await pool.connect();
+    await client.query(
+      `INSERT INTO tasks (title, description, user_id, priority, status) VALUES ($1, $2, $3, $4, $5);`,
+      [body.title, body.description, body.user_id, body.priority, body.status],
+    );
+    return { message: "Task created" };
+  };
+
+  static changeTaskStatusOrPriority = async (body) => {
+    const client = await pool.connect();
+    await client.query(
+      `UPDATE tasks SET priority=$1, status=$2 WHERE task_id=$3;`,
```

Comment:
When testing, if request only has `priority` then the status is set to `null`, same thing with `status`.
Please, just in case make it resistant to that (look up `COALESCE`).

Reply:
Fixed (good idea with this COALESCE didn;t hear about it).